fuzzy-native/src/score_match.h

```diff
@@ -7,6 +7,7 @@ struct MatchOptions {
   bool case_sensitive;
   bool smart_case;
   size_t max_gap;
+  std::string *root_path;
```

Comment:
No need for this optimization, since `MatchOptions` is only constructed once and just passed around by reference after that (just make it a regular value). Pointers are too tricky :P